packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanelSummary.tsx

```diff
@@ -339,171 +87,86 @@ export function TransferPanelSummary({
-                symbol: nativeCurrency.symbol
-              })}
-            </span>
+            </>
           )}
-        </div>
+          {formatAmount(
+            sameNativeCurrency ? estimatedTotalGasFees : estimatedL2GasFees,
+            {
+              symbol: nativeCurrency.symbol
+            }
+          )}{' '}
+          <NativeCurrencyPrice
+            amountInEth={estimatedTotalGasFees}
+            showBrackets
+          />
+        </span>
       </div>
 
-      {showBreakdown && (
-        <div className="flex flex-col space-y-2 text-sm text-gray-dark lg:text-base">
-          <div className="flex flex-row justify-between">
-            <div className="flex flex-row items-center space-x-2">
-              <span className="pl-4 font-light">{parentLayer} gas</span>
-              <Tooltip content={layerGasFeeTooltipContent(parentLayer)}>
-                <InformationCircleIcon className="h-4 w-4" />
-              </Tooltip>
-            </div>
-            <div className="flex w-2/5 flex-row justify-between tabular-nums">
-              <span className="font-light">
-                {formatAmount(estimatedL1GasFees, {
-                  symbol: parentChainNativeCurrency.symbol
-                })}
-              </span>
-
-              {showPrice && (
-                <span className="font-light">
-                  {formatUSD(ethToUSD(estimatedL1GasFees))}
-                </span>
-              )}
-            </div>
-          </div>
-          <div className="flex flex-row justify-between text-gray-dark">
-            <div className="flex flex-row items-center space-x-2">
-              <span className="pl-4 font-light ">{layer} gas</span>
-              <Tooltip content={layerGasFeeTooltipContent(layer)}>
-                <InformationCircleIcon className="h-4 w-4 " />
-              </Tooltip>
-            </div>
-            <div className="flex w-2/5 flex-row justify-between tabular-nums">
-              <span className="font-light">
-                {formatAmount(estimatedL2GasFees, {
-                  symbol: nativeCurrency.symbol
-                })}
-              </span>
-
-              {showPrice && (
-                <span className="font-light">
-                  {formatUSD(ethToUSD(estimatedL2GasFees))}
-                </span>
-              )}
-            </div>
-          </div>
-        </div>
-      )}
+      <div
+        className={twMerge(
+          'grid grid-cols-[260px_auto] items-center text-sm font-light tabular-nums text-gray-dark'
+        )}
+      >
+        <span>
+          You will receive on{' '}
+          {getNetworkName(isDepositMode ? childChain.id : parentChain.id)}:
+        </span>
+        <span className="font-medium">
+          {formatAmount(Number(amount))}{' '}
+          <TokenSymbolWithExplorerLink
+            token={token}
+            isParentChain={!isDepositMode}
+          />{' '}
+          {isTokenUSDC(token?.address) && isDepositMode && <>or USDC</>}
```

Comment:
NIT: This `USDC.e or USDC` clause is a bit confusing at first, we might need to add a little info popup here saying it will depend on the Bridging you select in the next step?

<img width="400" alt="image" src="https://github.com/OffchainLabs/arbitrum-token-bridge/assets/7558499/11c2c1b9-c5c8-4840-bd05-0d8a9b169eca">

Reply:
they will see the info on the popup when they click the button
i'd treat it as a separate UX issue rather than part of this PR